src/DataTable/DataTable.js

```diff
@@ -350,6 +349,7 @@ const DataTable = memo(({
                         key={column.id}
                         column={column}
                         sortIcon={sortIcon}
+                        sortable={!progressPending && data.length > 0}
```

Comment:
this is a little confusing when we also have `column.sortable` how about a rename this to `disabled`?